tests/ui/README.md

```diff
@@ -31,20 +31,20 @@ $ pip install -Ur requirements.txt
 ```
 
 ### Run the tests
-Tests are run using the command line. Below are a couple of examples of running the tests:
+Tests are run using tox on the command line. Below are a couple of examples of running the tests:
```

Comment:
By switching to Tox we can remove the "Create or activate a Python virtual environment" and "Install dependencies" sections.